index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```

Comment:
- [ ] Make sure you added these sections to your home page, to stick the original design

![Capture](https://user-images.githubusercontent.com/57408419/234873715-8595c101-dea0-4a74-886f-0aa5ce8fead0.JPG)